File: src/main/java/me/zimy/questionnaire/repositories/ResponderRepository.java
package me.zimy.questionnaire.repositories;

import me.zimy.questionnaire.domain.Responder;
import org.springframework.data.mongodb.repository.MongoRepository;
import org.springframework.stereotype.Repository;

/**
 * This is for usage with Spring Data JPA project
 *
 * @author Dmitriy &lt;Zimy&gt; Yakovlev
 * @since 12/4/14.
 */
@Repository
public interface ResponderRepository extends MongoRepository<Responder, Long> {
}

File: src/main/kotlin/me/zimy/questionnaire/repositories/ResponderRepository.kt
package me.zimy.questionnaire.repositories

import me.zimy.questionnaire.domain.Responder
import org.springframework.data.repository.Repository

/**
 * Created by zimy on 2/18/16.
 */
interface ResponderRepository : Repository<Responder, String> {
    fun findOne(id: String): Responder
    fun save(responder: Responder): Responder
}